Cargo.toml

```diff
@@ -15,6 +15,7 @@ readme = "README.md"
 codecov = { repository = "Smithay/calloop" }
 
 [dependencies]
+cfg-if = "1.0.0"
```

Comment:
Do we really need cfg if? Right now we're handling cases without it just fine. Not sure it worth?

Reply:
Sure, it was necessary when I split out the macos pipe vs. non-macos pipe2 implementations, but probably not any more.